packages/block-editor/src/components/block-inspector/style.scss

```diff
@@ -11,7 +11,8 @@
 	}
 
 	.components-base-control,
-	.components-radio-control {
+	.components-radio-control,
+	.block-editor-html-element-control {
```

Comment:
I hope we can eliminate these special classes in the future, but for now, it makes sense to group them.

@WordPress/gutenberg-components, do we have an issue to modernize block inspector controls styling now that most components have migrated to `__nextHasNoMarginBottom`?

Reply:
There is a suggestion to unify the layout of the block inspector. These styles should be removed in the future.